glomap/controllers/global_mapper.cc

```diff
@@ -209,6 +217,12 @@ bool GlobalMapper::Solve(const colmap::Database& database,
       if (ite != options_.num_iteration_bundle_adjustment - 1)
         run_timer.PrintSeconds();
 
+      // Normalize the structure
+      colmap::Reconstruction reconstruction;
+      ConvertGlomapToColmap(cameras, images, tracks, reconstruction, -1, true);
+      reconstruction.Normalize();
+      ConvertColmapToGlomap(reconstruction, cameras, images, tracks);
```

Comment:
This code block is identical for all 3 instances. Would it make sense to extract it into a helper function: `NormalizeReconstruction(...)` ? It's a little unfortunate that we need to go through the whole forward/backward conversion process each time instead of directly normalizing the glomap data structure?

Reply:
I do not have any quantitative numbers on the performance, but only qualitative ones: without rescaling, the number of points is 10282, and with normalization, the number of points is 12100 for one test set where the reconstruction scale is extreme after estimation. In this case, does it make sense to follow the COLMAP function and implement a new GLOMAP version? Then we don't need to do the model conversion at all